packages/frontend/src/ee/features/aiCopilot/components/ChatElements/ScrollToBottom.tsx

```diff
@@ -67,7 +67,11 @@ const ThreadScrollToBottom = ({
             checkCurrentScrollPosition: true,
             behavior: 'auto',
         });
-    }, [streamingState?.content, scrollToBottom]);
+    }, [
+        streamingState?.content,
+        streamingState.toolCalls.length,
```

Comment:
I guess that `toolCalls` can only change in size, but not in content? example: during streaming, this list contains tool A and B, but then B is replaced with C that has a lengthy name.
If that's the case, it might be better to just react on `toolCalls` changes (instead of length)

Reply:
correct, it can only be appended